modules/Kconfig.stm32

```diff
@@ -558,6 +558,12 @@ config USE_STM32_LL_DMA2D
 	  Enable STM32Cube Chrom-Art Accelerator™ controller (DMA2D) LL module
 	  driver
 
+config USE_STM32_LL_DMAMUX
```

Comment:
`USE_STM32_LL_FOO` are only defined for ll*.c files.
This does not exist in case of dmamux.

Reply:
static const struct dma_driver_api dmamux_funcs = 
dmamux has no api functions by itself, but requires a declaration of NULL functions for the DEVICE_API_AND_INIT. If using DEVICE_INIT, will make the return a NULL pointer of the device_get_binding((const char *)"DMAMUX_1");

Reply:
ok, removed from modules/Kconfig.stm32